mediafile.py

```diff
@@ -2101,12 +2101,68 @@ def as_dict(self):
         StorageStyle('ARTIST_CREDIT'),
         ASFStorageStyle('beets/Artist Credit'),
     )
+    artists_credits = ListMediaField(
```

Comment:
My reasoning for using plural was to implicitly reference the `artists` tag, which I believe should stay separate from `artist` because `artist` will include the 'join phrases', i.e. the & in "B.B. King & Eric Clapton", whereas `artists` will be a multi-tag and remove them.

I prefer the plural to keep it synonymous with the `artists` multi, but will let you make the final call :)

Reply:
Yeah, `artists_sort` makes sense to me for some reason! Maybe this should be `artists_credit` for symmetry?

Reply:
I like that. It's now `artists_credit`. Will update the beets PR to that name soon